conans/client/build/cmake.py

```diff
@@ -114,16 +115,34 @@ def flags(self):
     def is_multi_configuration(self):
         return is_multi_configuration(self.generator)
 
+    @property
+    def is_toolset_supported(self):
```

Comment:
Maybe better not create the properties, not necessary to expose this to consumers.

Reply:
okay, I'll remove it

Reply:
removed